RELEASE.md

```diff
@@ -0,0 +1,41 @@
+# Release
+Follow the steps outlined in this document to release a new version of `cloudgov-style`.
+
+### Blah blah
+- Releases should follow [semantic versioning](http://semver.org/).
+- The Github tags for the release should just be the number, no `v` before.
+- The project will be pushed to [npm](https://www.npmjs.com/package/cloudgov-style) and [RubyGems](https://rubygems.org/gems/cloudgov-style).
```

Comment:
Yes, so the logic is when there's a new tag, when a PR gets merged to master, travis will deploy.